.pre-commit-config.yaml

```diff
@@ -64,27 +64,26 @@ repos:
     args: ['-c', '"Google LLC"', '-l', 'apache']
     exclude: docs/videos/healthcare-and-life-sciences/lysozyme-example/submit.sh
     pass_filenames: true
+  - id: gocyclo
+    name: gocyclo
+    language: system
+    entry: gocyclo
+    args: ['-over', '15']
+    files: '\.go$'
```

Comment:
nit. instead of `files`, use:

```yaml
  types: [ go ]
```

See https://pre-commit.com/#filtering-files-with-types